main.py

```diff
@@ -29,10 +25,7 @@ def get_correct_currency():
 
 def fetch_data(url: str):
     response = requests.get(url)
-    try:
-        if response.status_code != 200:
-            raise MyError()
-    except MyError:
+    if response.status_code != 200:
         print(f"Unsuccessful response on request")
```

Comment:
Could we consider `logging` instead of `print()` https://docs.python.org/3/howto/logging.html#configuring-logging
In a simple way:
```
import logging


logger = logging.getLogger('simple_example')
```

```
logger.error("Unsuccessful response on request")  # replace print(f"Unsuccessful response on request")
```

Reply:
Yes of course we can.